### PR TITLE
User can search for national park

### PR DESCRIPTION
**Pull request by louis-zhangxa**
Screen Shot 2022-11-17 at 10 39 45 AM
Screen Shot 2022-11-17 at 10 39 55 AM

### CODE REVIEW

**Comment by louis-zhangxa** [comment]
https://user-images.githubusercontent.com/114542582/202531820-3f29ba7f-3984-43b8-944c-5700c8751a76.mov


https://user-images.githubusercontent.com/114542582/202531831-7457715d-a535-42a5-bb93-b7cf299ff429.mov

File: js/main.js [hunk 1]
```diff
@@ -21,6 +25,15 @@ function getRandomInt(max) {
   return Math.floor(Math.random() * max);
 }
 
+function submitSearch(event) {
+  event.preventDefault();
+  var mobileSearch = $mobileSearchContent.value;
+  var desktopSearch = $desktopSearchContent.value;
+  if (mobileSearch || desktopSearch !== '') {
+    return mobileSearch || desktopSearch;
```
Comment (codingcodymiller) [inline js/main.js]: Event handlers do not have return statements, since you can't access the return.

File: js/main.js [hunk 2]
```diff
@@ -21,6 +25,15 @@ function getRandomInt(max) {
   return Math.floor(Math.random() * max);
 }
 
+function submitSearch(event) {
+  event.preventDefault();
+  var mobileSearch = $mobileSearchContent.value;
+  var desktopSearch = $desktopSearchContent.value;
+  if (mobileSearch || desktopSearch !== '') {
```
Comment (codingcodymiller) [inline js/main.js]: In a `submit` event handler, the `event.target` is the `form` element that was submitted. You should use `event.target.elements` to access the value of the input in the form that was submitted, so you don't need this conditional. Make sure you give each `input` element a `name` attribute in the HTML with the same value so you can use the same property name to access either one.

File: js/main.js (hunk 1 above)
Comment (codingcodymiller) [inline js/main.js]: Shouldn't you be sending a request to get data using the search term? Where is that code?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by louis-zhangxa** [comment]
https://user-images.githubusercontent.com/114542582/202566140-4f277bd4-9279-4009-9c47-608d8299d57b.mov

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**